File: backend/src/main/java/com/maitaryane/um/response/TokenResponse.java
package com.maitaryane.um.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class TokenResponse{
	private String access_token;
	private String refresh_token;
	private String message;	
}

File: frontend/src/app/response/token-response.ts
export class TokenResponse {
    
    access_token : string;
    refresh_token : string;
    message : string;
    
    constructor(access_token : string, refresh_token : string, message : string)
    {
        this.access_token = access_token;
        this.refresh_token = refresh_token;
        this.message = message;
    }
}
